### PR TITLE
Soarian R4 CarePlan Documentation

### PR DESCRIPTION
**Pull request by stanlysampath**
Description
----
Request to review R4 Care Plan Documentation. 

PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
- [ ] 
CarePlan Home
CarePlan 2
CarePlan 3
CarePlan 4
CarePlan 5
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: layouts/soarian/r4/clinical.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<h1 id="clinical">Clinical</h1>
+
+<p>The Clinical category of HL7<sup>®</sup> FHIR<sup>®</sup> resources includes Summary, Diagnostics, Medications, Care Provision, Request and Response.</p>
```
Comment (HRNWriter) [inline layouts/soarian/r4/clinical.html:3]: Add a comma (.) after Request, so it's Request, and Response.
PRODOC-169058.

Comment (HRNWriter) [inline layouts/soarian/r4/clinical.html:3]: Add a comma (.) after Request, so the end of the sentence is:  Request, and Response.
PRODOC-169058.

**Review by HRNWriter — COMMENTED** [review]
See my comment with a minor change.
PRODOC-169058.

File: layouts/soarian/r4/clinical.html (hunk 1 above)
Comment (stanlysampath) [reply layouts/soarian/r4/clinical.html]: File update.

Comment (HRNWriter) [reply layouts/soarian/r4/clinical.html]: +1.

**Review by slvsridhar — APPROVED** [review]

**Review by ashu-bdvt — APPROVED** [review]

**Review by slvsridhar — APPROVED** [review]

**Review by VamsiMure — APPROVED** [review]
Approved changes

**Comment by slvsridhar** [comment]
After Publish :
image
image
image
image
image

**Pull request merged**